test/duct_main/test_e2e.py

```diff
@@ -6,7 +6,7 @@
 import pytest
 from con_duct.duct_main import SUFFIXES
 
-TEST_SCRIPT_DIR = Path(__file__).with_name("data")
+TEST_SCRIPT_DIR = Path(__file__).parent.parent / "data"
```

Comment:
Huh - I didn't even know `with_name` was a method...

Reply:
Yeah it's convenient but a little weird